src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs

```diff
@@ -95,7 +95,7 @@
    :render-workflow-details
    (fn [{:keys [props]} workflowId]
      (let [workflows (:workflows props)
-           workflowName (get-in workflows [0 "workflowEntity" "entityName"])]
+           workflowName (get-in (filterv #(= (get-in % ["workflowId"]) workflowId) workflows) [0 "workflowEntity" "entityName"])]
```

Comment:
Yeah, this is a little wonky.  But let's not refactor this now, it looks like more of a mess than it's worth.

I would prefer writing this line as:
```clojure
(as-> workflows $
      (filter #(= workflow-id (get % "workflowId")) $)
      (first $)
      (get-in $ "workflowEntity" "entityName"))
```
\<soapbox>If `:workflows` was keywordized, that could be:
```clojure
(->> workflows
     (filter (comp (partial = workflow-id) :workflowId))
     first :workflowEntity :entityName)
```
\</soapbox>